### ISSUE TITLE
Feature Request:  Show streaming information

### ISSUE THREAD

**Issue by akalongman** [4gray/iptvnator#192]
Would be great to show information about the stream, like resolution (e.g. 1248x1024), language (e.g. Georgian), fps (e.g. 30), etc.

**Comment by ericgl** [comment]
Yes, I would also like to see this info (resolution and frame-rate) of the stream.

**Comment by 4gray** [comment]
Shipped in v0.24.0 (#1578): open **Stream info** in the player overlay to see resolution, playback and source frame rate, stream and codec bitrates, audio details, buffer and dropped frames. It's available in the built-in web players and the frame-copy Embedded MPV engine; values a player can't report are hidden.
